Modules/_functoolsmodule.c

```diff
@@ -267,12 +355,28 @@ partial_vectorcall(partialobject *pto, PyObject *const *args,
         }
     }
 
-    /* Copy to new stack, using borrowed references */
-    memcpy(stack, pto_args, pto_nargs * sizeof(PyObject*));
-    memcpy(stack + pto_nargs, args, nargs_total * sizeof(PyObject*));
-
-    ret = _PyObject_VectorcallTstate(tstate, pto->fn,
-                                     stack, pto_nargs + nargs, kwnames);
+    Py_ssize_t nargs_new;
+    if (np) {
+        nargs_new = pto_nargs + nargs - np;
+        Py_ssize_t j = 0;   // Placeholder counter
```

Comment:
`j` implies an index; name it for what it is.

```suggestion
        Py_ssize_t placeholder_count = 0;
```

Reply:
it is an index of new args. However, placeholder_count is very suitable for name of `np`